default/stringtables/en.txt

```diff
@@ -4628,6 +4628,14 @@ Species Traits
 SPECIES_TRAITS_TEXT
 Non-exhaustive list of Species Traits:
 
+SYSTEM_SUPERIORITY_TITLE
+System Superiority
+
+SYSTEM_SUPERIORITY_TEXT
+'''When an armed fleet set to aggressive mode arrives at a system unoccupied by hostile fleets, it establishes system superiority. Once it has been established, supply is interrupted for hostile empires, industry of hostile planets can only be used on the planet it is generated on, and ground troop regeneration is stopped. Also, all starlane entries in the occupied system become guarded by the blockading force, so if hostile forces arrive, they can only exit by the starlane entry by which they arrived.
+If two hostile fleets arrive at an unoccupied system at the same time, no system superiority is established until only one side remains and the other one has either been destroyed or retreated. Until then, both empires will maintain supply for their empire.
```

Comment:
Also, the clause 
```so if hostile forces arrive, they can only exit by the starlane entry by which they arrived.```
is a bit incomplete/vague regarding what happens if multiple hostile fleets arrive along different starlanes, what if one gets destroyed in the meantime, etc.  I am pretty sure that at least one of those descriptions I linked had a few sentences about that.

Furthermore, the phrase 
```no system superiority is established``` 
should be clarified that it is with respect to each other -- they each can establish blockades against all other empires even while battling against each other.

Hmm, and the part about space monsters seems slightly incomplete also.    I really think that pretty much all of the information [in this explanation](http://www.freeorion.org/forum/viewtopic.php?p=76985#p76985) should be present, unless you think some of it has been superseded now?